server_src/runners/state.js

```diff
@@ -3212,9 +3217,9 @@ class Runner {
       });
       let updatePrinter = false;
       if (
-        settings.printer.printerName !== "" &&
-        settings.printer.printerName !==
-          farmPrinters[index].settingsAppearance.name
+        settings.printer.printerName.toString() !== "" &&
+        settings.printer.toString() !==
+          farmPrinters[index].settingsAppearance.name.toString()
```

Comment:
I think these we're why the update was been hooky as I was comparing object -> object.

Reply:
- `settings.printer.printerName` is the string to compare
- you're indeed now comparing an object `settings.printer` to string `farmPrinters[index].settingsAppearance.name` so that is object <-> string.
- please revert the lines where toString was added because it doesnt fix anything. It converts objects to [Object object] or something similar: not what you want at all.

These changes should be cancelled out on dev as this wont work, or does not improve the situation.